[0m15:26:20.346306 [debug] [MainThread]: Sending event: {'category': 'dbt', 'action': 'invocation', 'label': 'start', 'context': [<snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x000001D8F1066120>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x000001D8EED45D10>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x000001D8F1FFFED0>]}


============================== 15:26:20.352057 | 4cbc59fa-e660-482d-9cf7-754736281c10 ==============================
[0m15:26:20.352057 [info ] [MainThread]: Running with dbt=1.9.8
[0m15:26:20.353110 [debug] [MainThread]: running dbt with arguments {'printer_width': '80', 'indirect_selection': 'eager', 'log_cache_events': 'False', 'write_json': 'True', 'partial_parse': 'True', 'cache_selected_only': 'False', 'warn_error': 'None', 'version_check': 'True', 'debug': 'False', 'log_path': 'logs', 'fail_fast': 'False', 'profiles_dir': 'C:\\Users\\ASUS\\.dbt', 'use_colors': 'True', 'use_experimental_parser': 'False', 'empty': 'None', 'quiet': 'False', 'no_print': 'None', 'log_format': 'default', 'invocation_command': 'dbt init', 'warn_error_options': 'WarnErrorOptions(include=[], exclude=[])', 'introspect': 'True', 'target_path': 'None', 'static_parser': 'True', 'send_anonymous_usage_stats': 'True'}
[0m15:26:33.006744 [debug] [MainThread]: Starter project path: G:\Job_Project\Viettel\VDT_DE\mini_project\airbyte-airflow-data-ingestion\venv\Lib\site-packages\dbt\include\starter_project
[0m15:26:33.037288 [info ] [MainThread]: 
Your new dbt project "postgres_tranformations" was created!

For more information on how to configure the profiles.yml file,
please consult the dbt documentation here:

  https://docs.getdbt.com/docs/configure-your-profile

One more thing:

Need help? Don't hesitate to reach out to us via GitHub issues or on Slack:

  https://community.getdbt.com/

Happy modeling!

[0m15:26:33.038164 [info ] [MainThread]: Setting up your profile.
[0m15:28:36.998564 [info ] [MainThread]: Profile postgres_tranformations written to C:\Users\ASUS\.dbt\profiles.yml using target's profile_template.yml and your supplied values. Run 'dbt debug' to validate the connection.
[0m15:28:36.999654 [debug] [MainThread]: Command `dbt init` succeeded at 15:28:36.999519 after 136.96 seconds
[0m15:28:37.000010 [debug] [MainThread]: Sending event: {'category': 'dbt', 'action': 'invocation', 'label': 'end', 'context': [<snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x000001D8F3288180>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x000001D8F209D6D0>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x000001D8F1E4B240>]}
[0m15:28:37.000356 [debug] [MainThread]: Flushing usage events
[0m15:28:38.053649 [debug] [MainThread]: An error was encountered while trying to flush usage events
[0m15:40:06.731072 [debug] [MainThread]: Sending event: {'category': 'dbt', 'action': 'invocation', 'label': 'start', 'context': [<snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x000001C31B8E6120>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x000001C3195C5D10>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x000001C31C87FED0>]}


============================== 15:40:06.735200 | 9be08c00-56d7-43e6-ba4c-6abc85a23c21 ==============================
[0m15:40:06.735200 [info ] [MainThread]: Running with dbt=1.9.8
[0m15:40:06.736195 [debug] [MainThread]: running dbt with arguments {'printer_width': '80', 'indirect_selection': 'eager', 'write_json': 'True', 'log_cache_events': 'False', 'partial_parse': 'True', 'cache_selected_only': 'False', 'profiles_dir': 'C:\\Users\\ASUS\\.dbt', 'fail_fast': 'False', 'warn_error': 'None', 'log_path': 'logs', 'debug': 'False', 'version_check': 'True', 'use_colors': 'True', 'use_experimental_parser': 'False', 'empty': 'None', 'quiet': 'False', 'no_print': 'None', 'log_format': 'default', 'introspect': 'True', 'invocation_command': 'dbt debug', 'static_parser': 'True', 'target_path': 'None', 'warn_error_options': 'WarnErrorOptions(include=[], exclude=[])', 'send_anonymous_usage_stats': 'True'}
[0m15:40:06.765079 [info ] [MainThread]: dbt version: 1.9.8
[0m15:40:06.765779 [info ] [MainThread]: python version: 3.13.2
[0m15:40:06.766544 [info ] [MainThread]: python path: G:\Job_Project\Viettel\VDT_DE\mini_project\airbyte-airflow-data-ingestion\venv\Scripts\python.exe
[0m15:40:06.767129 [info ] [MainThread]: os info: Windows-11-10.0.26100-SP0
[0m15:40:06.839026 [info ] [MainThread]: Using profiles dir at C:\Users\ASUS\.dbt
[0m15:40:06.840049 [info ] [MainThread]: Using profiles.yml file at C:\Users\ASUS\.dbt\profiles.yml
[0m15:40:06.840721 [info ] [MainThread]: Using dbt_project.yml file at G:\Job_Project\Viettel\VDT_DE\mini_project\airbyte-airflow-data-ingestion\dbt_project.yml
[0m15:40:06.841343 [error] [MainThread]: Encountered an error:
Internal Error
  Profile should not be None if loading profile completed
[0m15:40:06.842278 [debug] [MainThread]: Command `dbt debug` failed at 15:40:06.842175 after 0.38 seconds
[0m15:40:06.842556 [debug] [MainThread]: Sending event: {'category': 'dbt', 'action': 'invocation', 'label': 'end', 'context': [<snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x000001C31DA5CFC0>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x000001C31DA529F0>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x000001C31DA8C160>]}
[0m15:40:06.842842 [debug] [MainThread]: Flushing usage events
[0m15:40:07.840326 [debug] [MainThread]: An error was encountered while trying to flush usage events
[0m15:40:43.144032 [debug] [MainThread]: Sending event: {'category': 'dbt', 'action': 'invocation', 'label': 'start', 'context': [<snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x0000023C409D6120>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x0000023C3E6B1D10>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x0000023C4196FED0>]}


============================== 15:40:43.148764 | 158988c3-8782-40f2-bf4b-a8d436e72235 ==============================
[0m15:40:43.148764 [info ] [MainThread]: Running with dbt=1.9.8
[0m15:40:43.149543 [debug] [MainThread]: running dbt with arguments {'printer_width': '80', 'indirect_selection': 'eager', 'log_cache_events': 'False', 'write_json': 'True', 'partial_parse': 'True', 'cache_selected_only': 'False', 'profiles_dir': 'C:\\Users\\ASUS\\.dbt', 'version_check': 'True', 'fail_fast': 'False', 'log_path': 'logs', 'warn_error': 'None', 'debug': 'False', 'use_colors': 'True', 'use_experimental_parser': 'False', 'no_print': 'None', 'quiet': 'False', 'empty': 'None', 'warn_error_options': 'WarnErrorOptions(include=[], exclude=[])', 'static_parser': 'True', 'log_format': 'default', 'invocation_command': 'dbt debug', 'target_path': 'None', 'introspect': 'True', 'send_anonymous_usage_stats': 'True'}
[0m15:40:43.171620 [info ] [MainThread]: dbt version: 1.9.8
[0m15:40:43.172522 [info ] [MainThread]: python version: 3.13.2
[0m15:40:43.172969 [info ] [MainThread]: python path: G:\Job_Project\Viettel\VDT_DE\mini_project\airbyte-airflow-data-ingestion\venv\Scripts\python.exe
[0m15:40:43.173435 [info ] [MainThread]: os info: Windows-11-10.0.26100-SP0
[0m15:40:43.236467 [info ] [MainThread]: Using profiles dir at C:\Users\ASUS\.dbt
[0m15:40:43.237120 [info ] [MainThread]: Using profiles.yml file at C:\Users\ASUS\.dbt\profiles.yml
[0m15:40:43.237556 [info ] [MainThread]: Using dbt_project.yml file at G:\Job_Project\Viettel\VDT_DE\mini_project\airbyte-airflow-data-ingestion\dbt_project.yml
[0m15:40:43.237939 [error] [MainThread]: Encountered an error:
Internal Error
  Profile should not be None if loading profile completed
[0m15:40:43.238603 [debug] [MainThread]: Command `dbt debug` failed at 15:40:43.238497 after 0.23 seconds
[0m15:40:43.238918 [debug] [MainThread]: Sending event: {'category': 'dbt', 'action': 'invocation', 'label': 'end', 'context': [<snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x0000023C41B6CFC0>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x0000023C41B629F0>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x0000023C41B98160>]}
[0m15:40:43.239307 [debug] [MainThread]: Flushing usage events
[0m15:40:44.207939 [debug] [MainThread]: An error was encountered while trying to flush usage events
[0m15:41:30.773675 [debug] [MainThread]: Sending event: {'category': 'dbt', 'action': 'invocation', 'label': 'start', 'context': [<snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x000001643A6E6120>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x00000164383C5D10>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x000001643B67FED0>]}


============================== 15:41:30.778075 | 626f5d0d-1948-42c2-8332-e2fc0d152787 ==============================
[0m15:41:30.778075 [info ] [MainThread]: Running with dbt=1.9.8
[0m15:41:30.779189 [debug] [MainThread]: running dbt with arguments {'printer_width': '80', 'indirect_selection': 'eager', 'log_cache_events': 'False', 'write_json': 'True', 'partial_parse': 'True', 'cache_selected_only': 'False', 'profiles_dir': 'C:\\Users\\ASUS\\.dbt', 'fail_fast': 'False', 'debug': 'False', 'log_path': 'logs', 'version_check': 'True', 'warn_error': 'None', 'use_colors': 'True', 'use_experimental_parser': 'False', 'no_print': 'None', 'quiet': 'False', 'empty': 'None', 'warn_error_options': 'WarnErrorOptions(include=[], exclude=[])', 'introspect': 'True', 'log_format': 'default', 'static_parser': 'True', 'target_path': 'None', 'invocation_command': 'dbt init', 'send_anonymous_usage_stats': 'True'}
[0m15:41:39.214206 [debug] [MainThread]: Starter project path: G:\Job_Project\Viettel\VDT_DE\mini_project\airbyte-airflow-data-ingestion\venv\Lib\site-packages\dbt\include\starter_project
[0m15:41:39.227910 [info ] [MainThread]: 
Your new dbt project "postgres_transformations" was created!

For more information on how to configure the profiles.yml file,
please consult the dbt documentation here:

  https://docs.getdbt.com/docs/configure-your-profile

One more thing:

Need help? Don't hesitate to reach out to us via GitHub issues or on Slack:

  https://community.getdbt.com/

Happy modeling!

[0m15:41:39.228770 [info ] [MainThread]: Setting up your profile.
[0m15:42:14.651051 [info ] [MainThread]: Profile postgres_transformations written to C:\Users\ASUS\.dbt\profiles.yml using target's profile_template.yml and your supplied values. Run 'dbt debug' to validate the connection.
[0m15:42:14.652425 [debug] [MainThread]: Command `dbt init` succeeded at 15:42:14.652236 after 44.02 seconds
[0m15:42:14.653023 [debug] [MainThread]: Sending event: {'category': 'dbt', 'action': 'invocation', 'label': 'end', 'context': [<snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x000001643B878180>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x000001643B71D5B0>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x000001643B4CB240>]}
[0m15:42:14.653702 [debug] [MainThread]: Flushing usage events
[0m15:42:15.793985 [debug] [MainThread]: An error was encountered while trying to flush usage events
